### PR TITLE
p4-b6-r3 Vatican

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+              De laude<!--line jumps down--> matrimo<!--line jumps down-->nii et ac<!--line jumps down-->ceptabilitate
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e121">
+              <lb ed="#C"/><ref xml:id="bHY6yh-d90266-Rd1e118">IN genesi capitulo 2</ref> <quote xml:id="bHY6yh-d90266-Qd1e127">Reliqu<lb ed="#C" break="no"/>et
+              homo patrem et matrem 
+              <lb ed="#C"/>et adherebit uxori sue et erunt 
+              <lb ed="#C"/>duo in carne</quote>
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e135">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e134">Tobias capitulo VI</ref> <quote xml:id="bHY6yh-d90266-Qd1e139">Hii qui
```
Comment (jeffreycwitt) [inline bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml:73]:
```suggestion
              ¶ <ref xml:id="bHY6yh-d90266-Rd1e134">Tobias capitulo VI</ref> <quote xml:id="bHY6yh-d90266-Qd1e139">hii qui 
```

File: bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e152">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e155">Aristotiles yconomicorum libro I</ref> <quote xml:id="bHY6yh-d90266-Qd1e156">Communicatio
+              <lb ed="#C"/>masculi et femine naturale est maxime</quote>
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e162">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e170">Iesus 
+              <lb ed="#C"/>filius sirach in ecclesiastico capitulo 25</ref> <quote xml:id="bHY6yh-d90266-Qd1e166">bonus qui habitat cum 
+              <pb ed="#C" n="125-v"/>
+              <cb ed="#B" n="a"/>
```
Comment (jeffreycwitt) [inline bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml:88]:
```suggestion
              <cb ed="#C" n="a"/>
```

File: bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            <p xml:id="bHY6yh-d90266-d1e184">
+              <ref xml:id="bHY6yh-d90266-Rd1e197"><unclear>Larolis</unclear> poeta 
+              <lb ed="#C"/>capitulo 4</ref> <quote xml:id="bHY6yh-d90266-Qd1e188">Est ne nouis nuptiis odit ue<lb ed="#C" break="no"/>nus 
+              atque parentum frustrantur gaudia 
+              <lb ed="#C"/>lacrymalis</quote>
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e196">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e211">Paulus I corinthios 7</ref> <quote xml:id="bHY6yh-d90266-Qd1e200">bonum est 
+              <lb ed="#C"/>homini mulierem tangere propter fornicationem 
+              <lb ed="#C"/>emin unusque suam habeat et unaqueque
```
Comment (jeffreycwitt) [inline bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml:106]:
```suggestion
              <lb ed="#C"/>enim uniusque suam habeat et unaque 
```

File: bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e241">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e260">Seneca de remediis fortuitorum capitulo 
+              <lb ed="#C"/>21 et finali</ref> <quote xml:id="bHY6yh-d90266-Qd1e245">uxor aduenticulum bonum 
+              <lb ed="#C"/>est nam est inter illa que unicuique 
+              <lb ed="#C"/>semel contingunt</quote>
+            </p>
+            <p xml:id="bHY6yh-d90266-d1e253">
+              ¶ <ref xml:id="bHY6yh-d90266-Rd1e274">Lex autentico titulo de con<lb ed="#C" break="no"/>sulibus 
+              § alia uero</ref> <quote xml:id="bHY6yh-d90266-Qd1e257">Vxores coruscantur
```
Comment (jeffreycwitt) [inline bHY6yh-d90266/cod-yyyiou_bHY6yh-d90266.xml:127]:
```suggestion
              § alia uero</ref> <quote xml:id="bHY6yh-d90266-Qd1e257">vxores coruscantur 
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Great job. especially with the line breaks at the beginning of the rubric. That's exactly how it should be done!

Just a few minor changes.

**Comment by GrantSchreib** [comment]
file corrected

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**